types/office-js/index.d.ts

```diff
@@ -2324,7 +2327,7 @@ declare namespace Office {
          * @param options Provides options to determine which event handler or handlers are removed.
          * @param callback Optional. A function that is invoked when the callback returns, whose only parameter is of type {@link Office.AsyncResult}.
          */
-        removeHandlerAsync(eventType: Office.EventType, options?: RemoveHandlerOptions, callback?: (result: AsyncResult) => void): void;
+        removeHandlerAsync(eventType: Office.EventType, options?: RemoveHandlerOptions, callback?: (result: AsyncResult<undefined>) => void): void;
```

Comment:
Why undefined? Do we know AsyncResult.value is always undefined when there's a callback to removeHandlerAsync?